packages/p/polyavatar.json

```
@@ -0,0 +1,36 @@
+{
+  "name": "polyavatar.js",
```

```suggestion
  "name": "polyavatar",
```
`name` should match the JSON file name

---

Agree. Thank you.